### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by jmcvey3**

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -20,19 +20,29 @@
...
+release = "0.2.10"
 
 
 # -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx_rtd_theme"]
+extensions = ["sphinx_rtd_theme",
```
Comment (maxwellevin) [inline docs/source/conf.py:35]: What do all these extra extensions do? Why is the autoapi extension disabled? –– I'm okay with it, but it's just not clear to me why this was done.

**Comment by maxwellevin** [comment]
We should also consider merging these changes into the `0.2.x` branch. I set up our readthedocs page to point to the `0.2.x` branch since we will soon have breaking changes on main for `v0.3.0`. With the `v0.3.0` release we will also want to update the docs again.

**Comment by maxwellevin** [comment]
What happened to the API Reference section? 

e.g., https://tsdat.readthedocs.io/en/latest/autoapi/index.html

**Comment by maxwellevin** [comment]
https://tsdat--38.org.readthedocs.build/en/38/

**Comment by jmcvey3** [comment]
> What happened to the API Reference section?
> 
> e.g., https://tsdat.readthedocs.io/en/latest/autoapi/index.html

I can add that back in.

File: docs/source/conf.py (hunk 1 above)
Comment (jmcvey3) [reply docs/source/conf.py:35]: Most of them have to do with reading and formatting docstrings for creating an API, aka the code source links in the "Configuring Tsdat" section. I took out the autoapi cause I've never found it very helpful for understanding code, but can put it back in.

Comment (maxwellevin) [reply docs/source/conf.py:35]: Ah, gotcha. Just a heads-up, I don't think those extensions are working correctly on readthedocs: https://tsdat--38.org.readthedocs.build/en/38/config/hook_functions.html

If you want get these working, or remove them, that's fine with me. But I would like the autoapi extension back in so we can have an API section in the sidebar – I find that useful.

Comment (maxwellevin) [reply docs/source/conf.py:35]: ^ Just saw your commit adding autoapi back in

**Comment by jmcvey3** [comment]
@maxwellevin Just rebased all of our commits together

**Pull request merged**